src/command_modules/azure-cli-iotcentral/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.1.3
```

Comment:
The version needs to be updated here: https://github.com/Azure/azure-cli/blob/6d79bdcd61bd99045ec325b0a903940573047f1e/src/command_modules/azure-cli-iotcentral/setup.py#L17

Reply:
Just noticed that--I'll fix